README.md

```diff
@@ -1,2 +1,2 @@
 # RPS
-rock, paper, scissors
+repositroy to learn introductory python development
```

Comment:
We'll make this readme more thorough and descriptive later. Things like: what the code actually does, how to set up a development environment, how to build and deploy, etc.

Great job opening your first PR!